diff -rupN pcw.dtsi.old pcw.dtsi
--- pcw.dtsi.old
+++ pcw.dtsi
@@ -44,11 +44,41 @@
 	is-decoded-cs = <0>;
 	num-cs = <3>;
 	status = "okay";
+	spidev@0 {
+		compatible = "ltc2488";
+		reg = <0x0>;
+		spi-max-frequency = <10000000>;
+	};
+	spidev@1 {
+		compatible = "ltc2488";
+		reg = <0x1>;
+		spi-max-frequency = <10000000>;
+	};
+	spidev@2 {
+		compatible = "ltc2488";
+		reg = <0x2>;
+		spi-max-frequency = <10000000>;
+	};
 };
 &spi1 {
 	is-decoded-cs = <0>;
 	num-cs = <3>;
 	status = "okay";
+	spidev@0 {
+		compatible = "ltc2488";
+		reg = <0x0>;
+		spi-max-frequency = <10000000>;
+	};
+	spidev@1 {
+		compatible = "ltc2488";
+		reg = <0x1>;
+		spi-max-frequency = <10000000>;
+	};
+	spidev@2 {
+		compatible = "ltc2488";
+		reg = <0x2>;
+		spi-max-frequency = <10000000>;
+	};
 };
 &uart0 {
 	cts-override ;
@@ -57,7 +87,8 @@
 	status = "okay";
 };
 &usb0 {
-	phy_type = "ulpi";
+	dr_mode = "host";
+	usb-phy = <&usb_phy0>;
 	status = "okay";
 };
 &clkc {
diff -rupN system-top.dts.old system-top.dts
--- system-top.dts.old
+++ system-top.dts
@@ -13,6 +13,15 @@
 		bootargs = "console=ttyPS0,115200 earlyprintk earlycon";
 		stdout-path = "serial0:115200n8";
 	};
+	usb_phy0: phy0 {
+		#phy-cells = <0>;
+		compatible = "usb-nop-xceiv";
+	};
+	pps {
+		compatible = "pps-gpio";
+		gpios = <&gpio0 54 0>;
+		capture-clear;
+	};
 	aliases {
 		ethernet0 = &gem0;
 		i2c0 = &i2c0;
